### DISCUSSION TITLE
Count from several tables

### DISCUSSION THREAD

**Discussion by cananda** [supabase/supabase#8880] [General]
I have several tables (8 of them) whose count needs to be displayed on the application (dashboard). What is the right way to approach this in supabase client?

**Comment by steve-chavez** [comment]
I'd suggest a function and calling it through [rpc](https://supabase.com/docs/reference/javascript/rpc).

```sql
create or replace function dashboard_counts() 
returns setof bigint as $$
  select count(*) as count1 from schema.tab1 
  union
  select count(*) as count2 from schema.tab2
  union
  select count(*) as count3 from schema.tab3
  -- .. other tables
$$ language sql stable;



```
